composer.json

```diff
@@ -17,7 +17,7 @@
         "intervention/imagecache": "^2.5",
         "laravel-notification-channels/telegram": "^4.0",
         "laravel-notification-channels/twitter": "^8.0",
-        "laravel/framework": "^10.0",
+        "laravel/framework": "~10.32.0",
```

Comment:
Hmm, we definitely don't want to do that. I see you say this is about Telegram env variables but I don't see how this relates to the framework version?

Reply:
It's because of this [bloc](https://github.com/laravel/framework/blob/5d9cf9ca81f5985b90dcbaa4a9a3f81c2f471e7a/src/Illuminate/Support/Testing/Fakes/NotificationFake.php#L311-L313). In our case it is empty when Telegram env variables are empty so it continues without count.

Reply:
Took me a bit but managed to resolve all of these issues.